checkers-backend/src/api/game/game.controller.ts

```diff
@@ -0,0 +1,73 @@
+import { Controller, Get, Post, Param, Body, Put, Delete } from '@nestjs/common';
+import { ApiService } from '../api.service';
+import { Datastore } from '@google-cloud/datastore';
+import { take } from 'rxjs';
+
+@Controller('api/v1/Game')
+export class GameController {
+    constructor(private readonly apiService: ApiService) {}
+    datastore = new Datastore({databaseId:'checkers-datastore', projectId:"checkers-zsl"});
+
+    @Get()
+    async FindAll(): Promise<object> {
```

Comment:
in typescript we should using camelcase, so are methods should beggin with lower-case
https://pl.wikipedia.org/wiki/CamelCase